src/proto/server/conn.rs

```diff
@@ -263,6 +263,19 @@ where
                         }
                     }
                 }
+                ReceivedPacket::RpcExecute(rpc) => {
+                    let state = state.lock().unwrap();
+                    let result = state.call_rpc(rpc.entry_id, rpc.parameter.clone());
+
+                    state.clients.iter().for_each(|(_, tx)| {
+                        tx.unbounded_send(Box::new(RpcResponse::new(
```

Comment:
RpcResponse should only be sent to the client that initiated the call

Reply:
fixed